src/scheduler/Scheduler.cpp

```diff
@@ -1012,7 +1012,29 @@ ExecGraphNode Scheduler::getFunctionExecGraphNode(unsigned int messageId)
 
     // Get the result for this message
     std::string statusKey = faabric::util::statusKeyFromMessageId(messageId);
+
+    // We want to make sure the message bytes have been populated by the time
+    // we get them from Redis. For the time being, we retry a number of times
+    // and fail if we don't succeed.
     std::vector<uint8_t> messageBytes = redis.get(statusKey);
+    int maxNumRetries = 3;
```

Comment:
This and the sleep time are magic numbers. As a rule, magic numbers should always be put into constants (i.e. a `#define` at the top of this file).